pages/settings/teams.tsx

```diff
@@ -1,46 +1,59 @@
+import { GetServerSideProps } from "next"
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
import { GetServerSideProps } from "next";
```